src/workflows/airqo_etl_utils/datautils.py

```
@@ -1072,6 +1097,77 @@
 
         return air_now_data
 
+    # ----------------------------------------------------------------------------------
+    # Storage
+    # ----------------------------------------------------------------------------------
+    def format_data_for_bigquery(
```

Ruff (0.8.2)

Redefinition of unused `format_data_for_bigquery` from line 400

(F811)